Sources/FlowStack/View+InteractiveDismiss.swift

```diff
@@ -214,8 +214,7 @@ class InteractiveDismissCoordinator: NSObject, ObservableObject, UIGestureRecogn
     private func update(offset: CGPoint, isEdge: Bool, hasEnded: Bool) {
         isUpdating = true
         onPan(offset)
-
-        let shouldDismiss = offset.y > threshold || (offset.x > threshold && isEdge)
+        let shouldDismiss = abs(offset.y) > threshold || (offset.x > threshold && isEdge)
```

Comment:
❓ Could we make this an option somehow? Maybe it's part of the FlowLink configuration?